zerver/lib/actions.py

```diff
@@ -1759,15 +1759,14 @@ def do_change_password(user_profile, password, commit=True,
                                  modified_user=user_profile, event_type='user_change_password',
                                  event_time=event_time)
 
-def do_change_full_name(user_profile, full_name, log=True):
-    # type: (UserProfile, Text, bool) -> None
+def do_change_full_name(user_profile, full_name):
+    # type: (UserProfile, Text) -> None
     user_profile.full_name = full_name
     user_profile.save(update_fields=["full_name"])
-    if log:
-        log_event({'type': 'user_change_full_name',
-                   'user': user_profile.email,
-                   'full_name': full_name})
-
+    event_time = timezone.now()
+    RealmAuditLog.objects.create(realm=user_profile.realm, acting_user=user_profile,
+                                 modified_user=user_profile, event_type='user_change_full_name',
+                                 event_time=event_time)
```

Comment:
I think it's possible for other users to change a user's name, so I think `acting_user` may need to be passed into this function as well.